config/locales/en/helpers.yml

```diff
@@ -128,21 +128,11 @@ en:
       cancel_check: Go back to check your answers
       results_page: Go to results page
     fieldset:
-      steps_conviction_conviction_subtype_form:
-        # default key
-        conviction_subtype: What type of order were you given?
-        # alternative keys defined with `i18_attribute`
-        referral_supervision_yro: What was your referral or youth rehabilitation order (YRO)?
-        custodial_sentence: What sentence were you given?
-        discharge: What discharge were you given?
-        financial: What were you ordered to pay?
-        military: What was your military conviction?
-        adult_military: What was your military conviction?
-        adult_motoring: What was your motoring conviction?
-        youth_motoring: What was your motoring conviction?
-        adult_community_reparation: What was your community, prevention or reparation order?
-        adult_discharge: What discharge were you given?
-        adult_custodial_sentence: What sentence were you given?
+      steps_check_under_age_form:
```

Comment:
`steps_check_under_age_form` and `steps_conviction_conviction_type_form` were again reintroduced after a bad merge. Need to be removed, these are already present in the `legend` section.